examples/datasets/a265032_1tc.32.txt

```diff
@@ -0,0 +1,71 @@
+p edge         32        68
```

Comment:
Nit: Perhaps move this to a subdirectory `dimacs/`?

Reply:
Done.